index.html

```diff
@@ -6,14 +6,71 @@
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
     </head>
-    <body>
-        <header>
-            <nav>
-                
-            </nav>
+    <body class="body" id="body">
```

Comment:
Yes, i use body id in script file to lock body when navigation menu is open (so that document's body doesn't scroll when i scroll menu). However i can delete class, cos i don't need it.